charts/router/templates/router-deployment.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   annotations:
     component.deis.io/version: {{ .Values.docker_tag }}
 {{- range $key, $value := .Values.deployment_annotations }}
-    {{ $key }}: {{ $value }}
+    {{ $key }}: "{{ $value }}"
```

Comment:
I think there's a `quote` function in sprig for this that might be safer it will escape any double quote that exist within the string to make it YAML safe.